### PR TITLE
Changes gold slime core hostile mob list to var

### LINKED ISSUE #11387: Gold Slime Cores are creating the base /simple_animal
https://github.com/tgstation/-tg-station/blob/master/code/modules/mob/living/simple_animal/simple_animal.dm

I know because I discovered one that was called animal and was invisible and chased its own tail.

### PR DESCRIPTION
**Pull request by Jordie0608**
Simple mobs are spawned by gold cores off the gold_core_spawnable var, 1 for hostile and 2 for friendly.

Also fixes a var in viscerators that was interpreted as a path thanks to glorious relative pathing.

Fixes #11387

### CODE REVIEW

**Comment by RemieRichards** [comment]
Make it a var that defaults to no.
Fuck this list, this list is ass.

**Comment by Cheridan** [comment]
Yeah, at this point it needs to be a var/flag and not a path list.

**Comment by bgobandit** [comment]
oh cool, I was going to put guardian spirits on the blacklist because they A) are common B) instantly die, but this works too

**Comment by KorPhaeron** [comment]
@bgobandit

They're already blacklisted

**Comment by bgobandit** [comment]
was it a recent change because I know I was getting them ingame a few days ago

**Comment by RemieRichards** [comment]
@Jordie0608 VARIABLE OR DEATH.

**Comment by Jordie0608** [comment]
Yeah yeah I'm going too, just busy and unable to atm.

**Comment by phil235** [comment]
This fixes #11387

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm [hunk 1]
```diff
@@ -132,8 +132,9 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 	mob_size = MOB_SIZE_TINY
-	flying
+	flying = 1
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: woops, my bad. I guess I worked a bit too fast when adding the flying var.

**Pull request merged**